doc/changelog/03-notations/12163-fix-12159.rst

```diff
@@ -0,0 +1,5 @@
+- **Fixed:**
+  Fix #12159 (Numeral Notations do not play well with multiple scopes
+  for the same inductive)
+  (`#12163 <https://github.com/coq/coq/pull/12163>`_,
+  by Pierre Roux).
```

Comment:
I think it's standard to link to the bug being fixed and mention it in the parenthetical
```suggestion
  Numeral Notations now play better with multiple scopes for the same
  inductive ( `#12163 <https://github.com/coq/coq/pull/12163>`_, fixes
  `#12159 <https://github.com/coq/coq/pull/12159>`_, by Pierre Roux).
```